R/deployApp.R

```diff
@@ -157,6 +155,25 @@ deployApp <- function(appDir = getwd(),
   check_directory(appDir)
   check_string(appName, allow_null = TRUE)
 
+  if (!is.null(appPrimaryDoc)) {
```

Comment:
Previously `appPrimaryDoc` was only checked in passing when determining `recordDir`.